v22.1/take-full-and-incremental-backups.md

```diff
@@ -14,10 +14,62 @@ There are two main types of backups:
 
 You can use the [`BACKUP`](backup.html) statement to efficiently back up your cluster's schemas and data to popular cloud services such as AWS S3, Google Cloud Storage, or NFS, and the [`RESTORE`](restore.html) statement to efficiently restore schema and data as necessary. For more information, see [Use Cloud Storage for Bulk Operations](use-cloud-storage-for-bulk-operations.html).
 
+{% include {{ page.version.version }}/backups/backup-to-deprec.md %}
+
 {{site.data.alerts.callout_success}}
  You can create [schedules for periodic backups](manage-a-backup-schedule.html) in CockroachDB. We recommend using scheduled backups to automate daily backups of your cluster.
 {{site.data.alerts.end}}
 
+## Backup collections
+
+ A _backup collection_ defines a set of backups and their metadata. The collection can contain multiple full backups and their subsequent [incremental backups](#incremental-backups). The path to a backup is created using a date-based naming scheme and stored at the [URI](backup.html#collectionURI-param) passed with the `BACKUP` statement.
+
+There are some specific cases where part of the collection data is stored at a different URI:
+
+- A [locality-aware backup](take-and-restore-locality-aware-backups.html). The backup collection will be stored according to the URIs passed with the `BACKUP` statement: `BACKUP INTO LATEST IN {collectionURI}, {localityURI}, {localityURI}`. Here, the `collectionURI` represents the default locality.
+- As of v22.1, it is possible to store incremental backups at a [different URI](#incremental-backups-with-explicitly-specified-destinations) to the related full backup. This means that one or multiple storage locations can hold one backup collection.
+
+By default, full backups are stored at the root of the collection's URI in a date-based path, and incremental backups are stored in the `/incrementals` directory. The following example shows a backup collection created using these default values, where all backups reside in one storage bucket:
+
+~~~
+Collection:
+|—— 2022
+  |—— 02
+    |—— 09-155340.13/
+      |—— Full backup files
+[...]
+|—— incrementals
+~~~
+
+[`SHOW BACKUPS IN {collectionURI}`](show-backup.html#view-a-list-of-the-available-full-backup-subdirectories) will display a list of the full backup subdirectories at the collection's URI.
+
+<a name="incremental-location-structure"></a> Alternately, the following directories also constitute a backup collection. There are multiple backups in two separate collection URIs. Each individual backup is a full backup and its related incremental backup(s). Despite using the [`incremental_location`](#incremental-backups-with-explicitly-specified-destinations) option to store the incremental backup in an alternative location, that incremental backup is still part of this backup collection as it depends on the full backup in the first cloud storage bucket:
```

Comment:
nit: in the first sentence, use 'separate URIs' instead of 'separate collection URIs' . A collection will always have only one collectionURI, but it may have multiple localityURis or explicit_incrementalURIs

Reply:
also, instead of "cloud storage bucket", use "collectionURI"?